tests/dags/test_pwdb_classifiers_dag.py

```diff
@@ -0,0 +1,21 @@
+from sem_covid.entrypoints.ml_dags.pwdb_classifiers_dag import DAG_NAME
+
+
+def test_pwdb_random_forest_experiment_dag_has_two_tasks_and_order(airflow_dag_bag):
+    dag = airflow_dag_bag.get_dag(dag_id=DAG_NAME)
+    assert dag is not None
+    tasks = dag.tasks
+    task_ids = list(map(lambda task: task.task_id, tasks))
+    assert {'feature_engineering', 'model_training'}.issubset(set(task_ids))
+
+    download_and_split_task = dag.get_task('feature_engineering')
```

Comment:
this testing can be automated, let's discuss what I mean. 
We can have a factory method that for each Pipeline creates assertions about the DAG structure